src/components/Wallet/WalletSelection.tsx

```diff
@@ -0,0 +1,182 @@
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import React, { useState } from 'react';
+import { FlatList, TouchableOpacity, StyleSheet } from 'react-native';
+import { useNavigation } from 'react-navigation-hooks';
+import { BigNumber } from 'bignumber.js';
+
+// Hooks
+import { useStableAssets, useNonStableAssets } from 'hooks/assets';
+import { useChainsConfig } from 'utils/uiConfig';
+
+// Components
+import ButtonGroup from 'components/layout/ButtonGroup';
+import { Spacing } from 'components/legacy/Layout';
+import TokenListItem from 'components/lists/TokenListItem';
+import Icon from 'components/core/Icon';
+import HorizontalProgressBar from 'components/Progress/HorizontalProgressBar';
+import ChainListItem from 'screens/Home/components/ChainListItem';
+
+// Constants
+import { TOKENS, STABLES } from 'constants/walletConstants';
+import { ASSET } from 'constants/navigationConstants';
+
+// Utils
+import { useThemeColors } from 'utils/themes';
+import { wrapBigNumberOrNil } from 'utils/bigNumber';
+import { buildAssetDataNavigationParam } from 'screens/Assets/utils';
+import { formatFiatValue } from 'utils/format';
+
+// Selector
+import { useSupportedChains } from 'selectors/chains';
+import { useFiatCurrency } from 'selectors';
+
+// Types
+import type { AssetCategoryRecordKeys } from 'models/AssetCategory';
+import type { TotalBalances } from 'models/TotalBalances';
+import type { Chain } from 'models/Chain';
+
+type Props = {
+  category: AssetCategoryRecordKeys;
+  accountTotalBalances: TotalBalances;
+  visibleBalance: boolean;
+};
+
+export default function ({ category, accountTotalBalances, visibleBalance }: Props) {
+  const [tabIndex, setTabIndex] = React.useState(0);
+  const [showMore, setShowMore] = useState(false);
+  const { tokens } = useStableAssets();
+  const chains = useSupportedChains();
+  const navigation = useNavigation();
+  const fiatCurrency = useFiatCurrency();
+
+  const chainsConfig = useChainsConfig();
+
+  const { tokens: nonStableTokens, percentage, totalPercentage } = useNonStableAssets();
+
+  const colors = useThemeColors();
+  let listOfAssets = tabIndex === 0 ? nonStableTokens : tokens;
+
+  const items = [
+    { key: TOKENS, title: TOKENS, component: null, color: colors.primaryAccent250 },
+    { key: STABLES, title: STABLES, component: null, color: colors.synthetic180 },
+  ];
+
+  const renderItem = (item: any) => {
```

Comment:
Is there an interface somewhere for `item`?

Reply:
Added here interface